server/redeemer.go

```diff
@@ -0,0 +1,398 @@
+package server
+
+// generate the go bindings for github.com/livepeer/go-livepeer/net/redeemer.proto first !
+//go:generate mockgen -source github.com/livepeer/go-livepeer/net/redeemer.pb.go -destination github.com/livepeer/go-livepeer/net/redeemer_mock.pb.go -package net
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"math/big"
+	gonet "net"
+	"net/url"
+	"sync"
+	"time"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/event"
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/livepeer/go-livepeer/eth"
+	"github.com/livepeer/go-livepeer/net"
+	"github.com/livepeer/go-livepeer/pm"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/peer"
+	"google.golang.org/grpc/status"
+)
+
+const rpcTimeout = 8 * time.Second
+
+var cleanupLoopTime = 1 * time.Hour
+
+// Redeemer is the interface for a ticket redemption gRPC service
+type Redeemer interface {
```

Comment:
Do we need to export this interface? Doesn't look like there are any users of this interface. If this is the case, we can just export the struct.

Reply:
fixed in 272bbfe493bfa2df89210b3a8dd193cd9d651bab